deploy/kubernetes/elastic-agent-standalone/elastic-agent-standalone-deployment.yaml

```diff
@@ -0,0 +1,58 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: elastic-agent
+  namespace: kube-system
```

Comment:
Is deploying in `kube-system` namespace what we used to do so far? I'd think this is a bit unusual.

Reply:
We do the same in the proposed Beat's manifests, ie: https://github.com/elastic/beats/blob/0616c04c204d7daf729eaaf45b8640ade2119772/deploy/kubernetes/metricbeat-kubernetes.yaml#L6